### ISSUE TITLE
composer exception M2.4.3

### ISSUE THREAD

**Issue by sugosol** [mageplaza/magento-2-smtp#333]
### Preconditions (*)

1. magento 2.4.3 open source

### Steps to reproduce (*)

1. composer require mageplaza/module-smtp

### Expected result (*)

1.  module installed

### Actual result (*)

Installation failed, reverting ./composer.json to its original content.
[Exception]                                                                                                                                   
Higher matching version 4.5.3 of mageplaza/module-smtp was found in public repository packagist.org                                            than 1.7.0 in private https://repo.magento.com. Public package might've been taken over by a malicious entity,   
please investigate and update package requirement to match the version from the private repository

**Comment by HuyPhuc98** [comment]
Hi,

Sorry for the late reply.
We will check this out, but to be able to install the SMTP module quickly please try to install the module manually by installing the SMTP module in the app/code/Mageplaza directory.
Then run the setup and deploy commands to activate the module.

Try it and let me know the results.
Thank you!

**Comment by paugnu** [comment]
Hi,

I've experienced exactly the same issue. Changing composer.json did work, but I don't think it's a good idea having this issue alive. Should you update Marketplace version? Remove it?

**Comment by uandiweb** [comment]
I agree with @paugnu. Either update the market place extension or just remove it from the market place. Changing Magento core files to overcome this issue is a bad idea. Thank you!

**Comment by fvlasie** [comment]
This issue is still occurring...

**Comment by attilanetlogiq** [comment]
Please fix this problem

**Comment by Ithalan** [comment]
Also experiencing this exact issue. Modifying the repo section in composer.json as @sugosol suggested got me past it, but it shouldn't be necessary to do so.

**Comment by attilanetlogiq** [comment]
Add **canonical = false** to your repository in composer.json
